index.js

```diff
@@ -1,293 +1,177 @@
 module.exports = {
+	root: true,
 	globals: {
-		'__DEV__': true,
-		'process': true,
-		'require': true,
-		'window': true,
-		'global': true,
-		'document': true,
-		'console': true,
-		'module': true,
-		'expect': true,
-		'clearInterval': false,
-		'clearTimeout': false,
-		'setInterval': false,
-		'setTimeout': false
+		__DEV__: true,
+		process: true,
+		window: true,
+		document: true,
 	},
 	env: {
-		'es6': true, // Enables ES6 globals
-		'jest': true
+		es6: true, // Enables ES6 globals
+		'shared-node-browser': true, // restrict to common globals to preserve isomorphic support
+		commonjs: true
 	},
-	extends: ['eslint:recommended', 'plugin:react/recommended'],
+	extends: ['eslint:recommended', 'plugin:react/recommended', 'plugin:react-hooks/recommended'],
+	ignorePatterns: [
+		'**/node_modules/**',
+		'**/build/**',
+		'**/dist/**',
+		'**/coverage/**'
+	],
 	parser: 'babel-eslint',
 	parserOptions: {
-		'ecmaVersion': 2018,
-		'sourceType': 'module',
-		'ecmaFeatures': {
-			'jsx': true
+		ecmaVersion: 2018,
+		sourceType: 'module',
+		ecmaFeatures: {
+			jsx: true
 		}
 	},
-	plugins: [
-		'react',
-		'react-hooks',
-		'babel',
-		'enact'
-	],
+	plugins: ['react', 'enact'],
 	settings: {
-		'react': {
+		react: {
 			'pragma': 'React',  // Pragma to use, default to "React"
 			'version': '15.0' // React version, default to the latest React stable release
```

Comment:
Should we change this?  It's kinda outdated!  Maybe this section can be removed entirely?

Reply:
Yepyep,  switching to `detect` is planned to be part of the next-step updates/new features once we know were good with refactoring existing rules in this first pass.